test/Nancy.Testing.Tests/QueryWrapperTests.cs

```diff
@@ -35,12 +40,11 @@ public void Should_filter_selection_when_second_indexer_invoked()
         {
             // Given
             var document =
-                CQ.Create(
-                    @"<html><head></head><body><table><tr><td></td><td></td></tr><tr><td></td><td></td></tr></body></html>");
+                this.parser.Parse(@"<html><head></head><body><table><tr><td></td><td></td></tr><tr><td></td><td></td></tr></body></html>");
 
-            var wrapper = new QueryWrapper(document);
+            var wrapper = new QueryWrapper(new[] { document.DocumentElement });
 
-            var row = wrapper["tr:first"];
+            var row = wrapper["tr:first-child"];

```

Comment:
How did this test pass in the firs place? `:first` will only return 1 element where `:first-child` should return 2 in this case.

Maybe we should put in an issue with CsQuery to state that `:first` was returning multiple elements?